todo.md

```diff
@@ -5,9 +5,11 @@ with definitions, translations, or whatever else you can come up with.
 ### Todo
 
 - [ ] Improve UI visuals
+- [ ] Add dark themes version of logo
 
 ### Backlog
 
+- [ ] Some regexs will be invalid (content_annotation_handler.js->createRegexMap will fail silently and annotation will not happen)
 - [ ] Set up build process to do dependency injection to handle different versions for different browsers.

```

Comment:
Is this still required? Specifically, is dependency injection still required?

Reply:
good point